README.rst

```diff
@@ -1,118 +1,70 @@
-  pip install -e .
-  python manage.py lms migrate eox_tagging --settings=<your app settings>
-
-Open edX with Tutor (Maple)
-----------------------------
-If you are using `Tutor <https://docs.tutor.overhang.io/gettingstarted.html>`_ to deploy the Open edX instance, please follow this steps:
-
-#. Install a new fresh instance of tutor following `this steps <https://docs.tutor.overhang.io/quickstart.html#quickstart-1-click-install>`_. *If you already have a tutor instance running you can skip this step.*
-#. Add to the Tutor configuration in the file ``cat "$(tutor config printroot)/config.yml"`` this lines that install eox-tagging and eox-core lib:
-    .. code-block:: yaml
-    
-        OPENEDX_EXTRA_PIP_REQUIREMENTS:
-        - eox_core
-        - eox_tagging
-#. Create a new tutor plugin that adds this settings to openedx common settings:
-    .. code-block:: yaml
-    
-        EOX_TAGGING_GET_ENROLLMENT_OBJECT = "eox_tagging.edxapp_wrappers.backends.enrollment_l_v1"
-        EOX_CORE_USERS_BACKEND = "eox_core.edxapp_wrapper.backends.users_m_v1"
-
-    .. note::
-        `Here <https://github.com/eduNEXT/eox-tagging/issues/83>`_ is an example of creating a tutor plugin with a yaml file. 
-        But there are other ways to create tutor plugins, please go to tutor docs and see how to add settings in Open edX common settings.
-#. Install the plugin doing ``tutor plugins enable eox-tagging-plugin`` and then ``tutor config save``.
-#. Build the openedx image doing ``tutor images build openedx``.
-#. Start the tutor instance with the new config doing ``tutor local quickstart``.
-#. All set to use eox-tagging lib!
+   .. code-block:: yaml
+      
+      OPENEDX_EXTRA_PIP_REQUIREMENTS:
+         - eox-tagging=={{version}}
+         
+#. Save the configuration with ``tutor config save``.
+#. Build the image and launch your platform with ``tutor local launch``.
 
 Compatibility Notes
 --------------------
 
-+-------------------+----------------+
-| Open edX Release  |     Version    |
-+===================+================+
-|      Ironwood     |   < 0.9 < 3.0  |
-+-------------------+----------------+
-|       Juniper     |   >= 0.9 < 3.0 |
-+-------------------+----------------+
-|        Koa        |   >= 2.2 < 5.0 |
-+-------------------+----------------+
-|       Lilac       |   >= 2.2 < 5.0 |
-+-------------------+----------------+
-|       Maple       | >= 4.0 < 6.0   |
-+-------------------+----------------+
-|       Nutmeg      |  >= 5.0 < 6.0  |
-+-------------------+----------------+
-|       Olive       |  >= 5.0 < 6.0  |
-+-------------------+----------------+
-|       Palm        |      >= 6.0    |
-+-------------------+----------------+
-|      Quince       |      >= 7.0    |
-+-------------------+----------------+
-
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
-
-**Ironwood**
```

Comment:
@dcoa why are we removing the listed settings for the previous releases and only keeping the latest? 
Could you please explain this decision and the reasoning behind it in the cover letter?

Reply:
This has been discussed and clarified in https://github.com/eduNEXT/eox-tenant/pull/210.